components/textfield/index.css

```diff
@@ -10,414 +10,179 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "./themes/express.css";
-
 .spectrum-Textfield {
-  /* set input line-height to the height of the textfield - prevents the cutting off of diacritics in some languages */
-  /* disallow mod for max compatibility */
-  --spectrum-textfield-input-line-height: var(--spectrum-textfield-height);
-  --spectrum-texfield-animation-duration: var(
-		--spectrum-animation-duration-100
-	);
-
-	--spectrum-textfield-width: 240px; /* override per api */
-	--spectrum-textfield-min-width: var(
-		--spectrum-text-field-minimum-width-multiplier
-	);
-	--spectrum-textfield-corner-radius: var(--spectrum-corner-radius-100);
+	/* set input line-height to the height of the textfield - prevents the cutting off of diacritics in some languages */
+	/* disallow mod for max compatibility */
+	--spectrum-textfield-input-line-height: var(--spectrum-textfield-height);
```

Comment:
I did look into this, as this is _**not**_ the line height seen in Figma. Using the line height token from Figma would cause [the original bug that was the reason this line height was added](https://github.com/adobe/spectrum-css/pull/2123), so this has intentionally been left as is.